docs/pages/qa/multimodal/embedding_cache.md

```
@@ -0,0 +1,64 @@
+  &> logs/dynamo_vllm_e_pd_embedding_cache_disabled.txt &
+```
+
+```
+bash examples/backends/vllm/launch/disagg_multimodal_e_pd.sh \
+  --multimodal-embedding-cache-capacity-gb 10 \
+  &> logs/dynamo_vllm_e_pd_mm_embedding_cache_enabled.txt &
+```
+
+## TRT-LLM
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)